src/DynamoCore/Graph/Nodes/CodeBlockNode.cs

```diff
@@ -86,6 +86,14 @@ public override bool IsInputNode
             get { return false; }
         }
 
+        /// <summary>
+        ///     Indicates whether node is an output node
```

Comment:
can we change this comment to help devs remember this is related to the presence of the UI checkbox?

Reply:
Ok.  Shall I fix it everywhere for IsInputNode and IsOutputNode?